File: src/apps/uploadServer.ts
import express from "express";
import multer from "multer";
import { extname } from "path";

const uploadServer = express();

const storage = multer.diskStorage({
  destination: "./uploads",
  filename: (_req, file, fn) =>
    fn(null, `${Date.now()}${extname(file.originalname)}`),
});

const upload = multer({ storage: storage });

uploadServer.put("/upload-receive", upload.single("someFile"), (req, res) => {
  console.log(req.file, req.body);

  res
    .status(200)
    .type("application/json")
    .send({ result: "Everything is fine" });
});

export { uploadServer };

File: src/apps/py/uploadServer.py
from flask import Flask, request
from flask_cors import CORS, cross_origin

app = Flask(__name__)
cors = CORS(app)

@app.route('/upload-receive', methods=['PUT'])
def upload_file():
    print(request.headers)

    uploaded_file = request.files['someFile']
    if uploaded_file.filename != '':
        uploaded_file.save("../../../uploads/" + uploaded_file.filename)
        return {
            "result": "Everything is fine"
        }, 200
    
    return {
        "result": "Not ok"
    }, 500

if __name__ == '__main__':
  app.run(host='0.0.0.0', port=3002)